app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/patchManifestConfig.controller.ts

```diff
@@ -84,4 +97,42 @@ export class KubernetesV2PatchManifestConfigCtrl implements IController {
   public handleManifestSelectorChange = (): void => {
     this.$scope.$applyAsync();
   };
+
+  public getMergeStrategy = (): MergeStrategy => {
```

Comment:
if this method is only called within this controller, the access modifier can be `private`